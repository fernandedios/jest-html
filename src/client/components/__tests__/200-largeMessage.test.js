/* eslint-env jest */
/* eslint-disable global-require, import/newline-after-import */
import React from 'react';
import renderer from 'react-test-renderer';
import LargeMessage from '../200-largeMessage';

// https://github.com/facebook/react/issues/7386#issuecomment-238091398
jest.mock('react-dom');

describe('LargeMessage', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <LargeMessage>
          An error has occurred:<br />
          Undefined is not a function!
        </LargeMessage>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
