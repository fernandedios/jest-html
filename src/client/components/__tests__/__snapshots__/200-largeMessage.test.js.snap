// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LargeMessage renders correctly 1`] = `
<div
  className="giu-large-message"
  style={
    Object {
      "color": "#999999",
      "fontFamily": "sans-serif",
      "fontSize": "1.4em",
      "fontWeight": 100,
      "padding": "0.8em",
      "textAlign": "center",
    }
  }
>
  An error has occurred:
  <br />
  Undefined is not a function!
</div>
------------HTML PREVIEW---------------
<div
  class="giu-large-message"
  style="color:#999999;font-family:sans-serif;font-size:1.4em;font-weight:100;padding:0.8em;text-align:center;"
>
  An error has occurred:
  <br>
  Undefined is not a function!
</div>
`;
